draft-ietf-privacypass-rate-limit-tokens.md

```diff
@@ -922,6 +957,40 @@ struct {
 } Token;
 ~~~
 
+## Penalization of Invalid Clients and Issuers {#penalization}
```

Comment:
Maybe worth splitting into two sections (one for client and one for issuers) or at least subsections?

Reply:
I'd prefer to keep these together, since some of the penalizations apply to both/either